ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -173,7 +173,7 @@ const mapStateToProps = (state, ownProps) => {
   const transactionData = parseStandardTokenTransactionData(data);
   const tokenToAddress = getTokenAddressParam(transactionData);
 
-  const { balance } = accounts[fromAddress];
+  const { balance } = accounts[fromAddress] || {};
```

Comment:
The defined type for a `Numeric`, where this `balance` value ends up does not allow `undefined`. Can you fix the type or this value?

Reply:
thanks David! 

This appears to be a multichain issue after reading what @sleepytanya reported below. And following your comment, I think we should aim to preserve the type here and restore this behavior which expects transaction data

Looking into this some more and will ping the multichain team to see if they have any leads

Reply:
I don't think anything specific to multichain would cause a `fromAddress` or `transactionData` to go missing.  If those don't exist, I would think the transaction confirmation should just not be rendered.  Could this have something to do with transaction simulation?  Multichain really shouldn't affect anything here.

Reply:
interesting. This is something to consider. Ty @darkwing!

as @sleepytanya mentioned below, we will create a separate ticket to investigate this issue further. 

In the meantime, in case we'd like to merge this, I've updated the PR to preserve the `Numeric` state and send an error to Sentry when this issue occurs

Reply:
to follow up here, the Wallet UX team found the lingering issue reported below was an expected behavior by how the code was written. Only eth_signTypedData_v4 and personal_sign signatures were set to update to the selected network. Other signature types do not need to enforce the chainId / network in order for the signature to be signed. The plan is to allow other signatures to behave the same as eth_signTypedData_v4 and personal_sign during network switches to adopt consistent UI behavior

Reply:
has @davidmurdoch's comment regarding the type been addressed?

Reply:
hmm sorry for the confusion and thanks for the comment here! I meant to double-check this.

so the `subtractHexes` and `isBalanceSufficient` methods below expect balance to be a string type. I think setting the value in these cases to `0x0` should be what we expect